optimize_sidebars.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
           "Resource authorizations",
           "components/concepts/resource-authorizations/"
         ),
+        docsLink("User restrictions", "components/concepts/user-restrictions/"),
```

Comment:
This is the SaaS page that does not yet exist, so the link would be dead I believe